### PR TITLE
Migrate from Carbon and add support for built-in accounts

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Carbon.ScheduledTasks/prism.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"PSModules": [
```
Comment (KhoiKy) [inline Carbon.ScheduledTasks/prism.json]: Indentation looks off.

**Review by KhoiKy — APPROVED** [review]

File: Carbon.ScheduledTasks/prism.lock.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "PSModules": [
+    {
+      "name": "Carbon.Accounts",
+      "version": "1.0.0-rc5",
```
Comment (pnstickne) [inline Carbon.ScheduledTasks/prism.lock.json]: If not rc, feels awkward to allow/lock downstream rcs.

**Review by pnstickne — APPROVED** [review]

File: Carbon.ScheduledTasks/Carbon.ScheduledTasks.psm1 [hunk 3]
```diff
@@ -12,20 +12,166 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
+using namespace System.Collections.Generic;
```
Comment (pnstickne) [inline Carbon.ScheduledTasks/Carbon.ScheduledTasks.psm1:15]: Is convention of using before/after #Requires and Set-StrictMode? Those almost feel 'more' important.

File: Carbon.ScheduledTasks/prism.lock.json (hunk 2 above)
Comment (pnstickne) [reply Carbon.ScheduledTasks/prism.lock.json]: Perhaps if there is a way to prism.lock like 'use rc only if this is rc' - although how to get that inversion 🩰

File: prism.json [hunk 4]
```diff
@@ -1,4 +1,8 @@
 {
 	"PSModules": [
```
Comment (pnstickne) [inline prism.json:2]: Indent

File: Carbon.ScheduledTasks/Carbon.ScheduledTasks.psm1 (hunk 3 above)
Comment (splatteredbits) [reply Carbon.ScheduledTasks/Carbon.ScheduledTasks.psm1:15]: PowerShell requires it be one of the first lines in the file, excluding comments.

**Pull request merged**